packages/contracts/src/connect-contracts.ts

```diff
@@ -0,0 +1,68 @@
+import { Signer, Contract, providers } from 'ethers'
+import { getL1ContractData, getL2ContractData } from './contract-data'
+
+export const connectL1Contracts = async (
+  signerOrProvider: Signer | providers.Provider,
+  network: 'goerli' | 'kovan' | 'mainnet'
+): Promise<{ [key: string]: Contract }> => {
```

Comment:
We should be typing this object e.g., https://github.com/graphprotocol/common-ts/blob/master/packages/common-ts/src/contracts/index.ts#L34